taxonomies/additives.txt

```diff
@@ -1487,7 +1487,7 @@ oc:E150, caramèl
 pl:E150, karmel
 pt:E150, caramelo
 ro:E150, Carmel
-ru:E150, карамель
+ru:E150, карамель, burnt sucrose
```

Comment:
the translations should only be in the target language, why add "burnt sucrose"?
and it looks like we have no product with it: https://world.openfoodfacts.org/ingredient/burnt-sucrose

have you seen it in an actual product?

Reply:
Oops, looks like a mistake